addons/web/static/src/core/frequent_emoji_service.js

```diff
@@ -0,0 +1,32 @@
+import { reactive } from "@odoo/owl";
```

Comment:
suggestion: create a folder /core/emojis/ to gather emojis stuff

Reply:
side question: should this be in web? I guess it is for a reason, but who knows

Reply:
I'll create the folder. The emoji picker is in web and keeps track of the frequent emojis so it should be in web as well.

![image](https://github.com/user-attachments/assets/70b7e120-1c14-4be1-a49d-3246e56cac24)